test/ui/test/button-link-edit-browse.jsx

```diff
@@ -0,0 +1,59 @@
+import ButtonLinkEditBrowse from '../../../src/components/buttons/button-link-edit-browse.jsx';
+
+(function() {
+	'use strict';
+
+	var assert = chai.assert;
+	var Simulate = ReactTestUtils.Simulate;
+	var TestUtils = ReactTestUtils;
+	var documentBrowseLinkCallback;
+	var documentBrowseLinkUrl = 'http://alloyeditor.com/some/location';
+
+	describe('ButtonLinkEditBrowse', function() {
+		before(function(done) {
```

Comment:
Arrow function would be good here (and throughout).

Reply:
Will use ES6+ ;) Thanks